internal/fs/wrappers/monitoring.go

```diff
@@ -46,6 +46,26 @@ var (
 	tracer = otel.Tracer(name)
 )
 
+// Error categories
+const (
+	deviceErr         = "DEVICE_ERROR"
+	dirNotEmptyErr    = "DIR_NOT_EMPTY"
+	fileExistsErr     = "FILE_EXISTS"
+	fileDirErr        = "FILE_DIR_ERROR"
+	notImplementedErr = "NOT_IMPLEMENTED"
+	ioErr             = "IO_ERROR"
+	intrErr           = "INTERRUPT_ERROR"
```

Comment:
we can keep the variable names normal instead of shortened -> intrErr = interruptError or nwErr as networkError